internal/statestore/redis.go

```diff
@@ -456,6 +456,10 @@ func (rb *redisBackend) FilterTickets(ctx context.Context, filters []*pb.Filter,
 
 // UpdateAssignments update the match assignments for the input ticket ids
 func (rb *redisBackend) UpdateAssignments(ctx context.Context, ids []string, assignment *pb.Assignment) error {
+	if assignment == nil || (assignment.Connection == "" && assignment.Error == "" && assignment.Properties == "") {
```

Comment:
Should we care about this?

Reply:
I think we do need the nil check, otherwise calling this method with nil assignment will panic. What do you think for the content checks?